MapboxNavigation/NavigationViewController.swift

```diff
@@ -266,16 +266,12 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
     /**
      Shows a button that allows drivers to report feedback such as accidents, closed roads,  poor instructions, etc. Defaults to `false`.
      */
-    public var showsReportFeedback: Bool = false {
-        didSet {
-            mapViewController?.reportButton.isHidden = !showsReportFeedback
```

Comment:
Note that this `didSet` made it possible to show or hide the feedback button on the fly.

Reply:
Eh, I don't really see the use case for this.